lib/index.js

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+/* global webpage, window, document */
+var bedrock = require('bedrock');
+var cache = require('bedrock-redis');
+var config = bedrock.config;
+var express = require('bedrock-express').express;
+var phridge = require('phridge');
+var prerender = require('prerender-node');
+var querystring = require('querystring');
+var url = require('url');
+
+require('bedrock-requirejs');
+require('bedrock-views');
+
+var api = {};
+module.exports = api;
+
+require('./config');
+
+var logger = bedrock.loggers.get('app');
+var phantom = null;
+var requestQueue = {};
+
+var args = {
+  '--load-images': false,
+  '--ignore-ssl-errors': true,
+  '--ssl-protocol': 'tlsv1.2'
```

Comment:
Sure, we can make a config var to hold those defaults and allow for modifications.